spec/index.html

```diff
@@ -365,19 +368,25 @@ <h3>Triple Terms and Reification</h3>
       </figcaption>
     </figure>
 
-    <p class="note">
-      Since <a>triple terms</a> denote abstract, logical <a>propositions</a>,
-      they are only allowed as objects of other triples.
-      Assertions involving them should commonly use <a>reifying triples</a>;
-      the <a>reifier</a> is then used for further description.
-      There can be multiple, distinct reifiers related to the same abstract
-      proposition, such as statements with different sources, or circumstances
-      with different characteristics.
-      One reifier may also be used to reify multiple, distinct propositions,
-      expressing different propositional aspects using the same reifier.
-      Since a proposition so reified does not have to hold, it is possible to make
-      statements about any kind of statements, including unasserted contradictions.
-    </p>
+    <div class="note">
+      <p><a>Triple terms</a> always denote abstract, logical <a>propositions</a>,
+        while <a>reifiers</a> may denote a variety of things that are related
+        to these propositions (such as a statement or belief that the
+        proposition holds, or an event or circumstance that makes the
+        proposition true).
+        It is therefore expected that triple terms are commonly used as
+        <a>objects</a> of <a>reifying triples</a>, and that the reifiers
+        (rather than the triple terms) are used in further descriptions.</p>
+      <p>
+        There can be multiple, distinct reifiers related to the same abstract
+        proposition, such as statements with different sources, or
+        circumstances with different characteristics. One reifier may also be
+        used to reify multiple, distinct propositions, expressing different
+        propositional aspects using the same reifier.</p>
+      <p>Since a proposition so reified does not have to hold, it is possible
+        to make statements about any kind of statements, including unasserted
+        contradictions.</p>
```

Comment:
```suggestion
      <p>Since a proposition so reified does not have to hold, it is possible
        to make statements about any kind of statement, including an unasserted
        statement that contradicts another statement, whether asserted or not.</p>
```